.eslintrc.json

```diff
@@ -1,17 +1,24 @@
 {
 	"root": true,
 	"parser": "@typescript-eslint/parser",
+	"parserOptions": {
+		"project": "tsconfig.eslint.json"
+	},
 	"plugins": ["@typescript-eslint", "jest", "tsdoc"],
 	"extends": [
 		"eslint:recommended",
 		"plugin:@typescript-eslint/recommended",
+		"plugin:@typescript-eslint/recommended-requiring-type-checking",
+		"airbnb-typescript/base",
```

Comment:
Why add this?

Reply:
The first one enables the rules that require type checking; they're disabled by default because it takes extra setup, but really it's supposed to be included. Practically, it mostly prevents you from using `any` in bad ways.